libs/sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -2037,7 +2058,7 @@ private static void printStatement(String sql) {
             statement instanceof GrantPrivilege ||
             statement instanceof DenyPrivilege ||
             statement instanceof RevokePrivilege ||
-            statement instanceof DropUser ||
+            statement instanceof DropRole ||
             statement instanceof DropAnalyzer ||
             statement instanceof DropFunction ||
```

Comment:
Could you add `AlterRole` as well? Looks like `AlterUser` is not included here on master.

Reply:
thx! please check again

Reply:
test failures look related

Reply:
yep, sorry, fixed.